app/src/main/java/com/example/karam/LandingPage.java

```diff
@@ -0,0 +1,82 @@
+package com.example.karam;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListView;
+import android.widget.Toast;
+
+public class LandingPage extends AppCompatActivity {
+    private static String[] languages = {"English",
+            "Hindi",
+            "Marathi"};
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.landing_page);
+        ArrayAdapter<String> adapter = getAdapterWithLanguages();
```

Comment:
combine line 25 and 27

Reply:
How? Line 27 uses variable defined in line 26.